Packs/Azure-Enrichment-Remediation/ReleaseNotes/1_1_9.md

```diff
@@ -0,0 +1,8 @@
+
+#### Playbooks
+
+##### Azure - User Investigation
+
+- Added tasks to check if the Azure Sentinel tables are available for querying.
```

Comment:
added tasks or a task?

Reply:
Tasks. 
New task to query the available tables and new conditional tasks to check whether to query each table based on the output of the available table query.